modules/oracle-xe/src/main/java/org/testcontainers/containers/OracleContainer.java

```diff
@@ -102,14 +145,33 @@ public OracleContainer withPassword(String password) {
         return self();
     }
 
+    @Override
+    public OracleContainer withDatabaseName(String databaseName) {
+        if (StringUtils.isEmpty(databaseName)) {
+            throw new IllegalArgumentException("Database name cannot be null or empty");
+        }
+
+        if (ORACLE_DEFAULT_DBS.contains(databaseName.toLowerCase())) {
```

Comment:
We could simpify this to `DEFAULT_DATABASE_NAME.equals(databaseName.toLowerCase())`, couldn't we? As-is looks like it's future-proofing, but for now `ORACLE_DEFAULT_DBS` is just a constant list with one value, so it doesn't feel like we need to do this just yet.